lib/python-indent.js

```diff
@@ -294,4 +294,43 @@ export default class PythonIndent {
         // Set the indent
         this.editor.setIndentationForBufferRow(row, indent);
     }
+
+    reindent() {
+        this.editor = atom.workspace.getActiveTextEditor();
+
+        if (!this.editor ||
```

Comment:
Can you move this and the same code for halting in `PythonIndent.indent` out into a separate function?